packages/template-typescript-minimal/app/components/_app-config/index.js

```diff
@@ -4,9 +4,43 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import React from 'react'
+import PropTypes from 'prop-types'
 import {withLegacyGetProps} from '@salesforce/pwa-kit-react-sdk/ssr/universal/components/with-legacy-get-props'
 import {withReactQuery} from '@salesforce/pwa-kit-react-sdk/ssr/universal/components/with-react-query'
-import AppConfig from '@salesforce/pwa-kit-react-sdk/ssr/universal/components/_app-config'
+import {getAppOrigin} from '@salesforce/pwa-kit-react-sdk/utils/url'
+import {CommerceApiProvider} from '@salesforce/commerce-sdk-react'
+
+const AppConfig = ({children, locals = {}}) => {
+    const appOrigin = getAppOrigin()
+    return (
+        <CommerceApiProvider
```

Comment:
I plan to revert changes made to the files in the typescript-minimal. They're only temporary for our own testing.